docs/content/troubleshooting.md

```diff
@@ -0,0 +1,71 @@
+
+------
+
+**Q:** I’m getting the following compilation error when building the GDK: `Error C2248: FRepLayout::Cmds': cannot access private member declared in class 'FRepLayout`.
+
+**A:** You're building against an unsupported version of Unreal Engine. Make sure you're targeting the fork of Unreal Engine that the GDK requires. See the [setup guide](../setup-and-installing.md#building) for more details.
+
+------
+
+**Q:** When running the Interop Code Generator, I’m getting the following error: `Error: Could not move generated interop files during the diff-copy stage`.
+
+**A:** Due to the [known issue](../known-issues.md) with piped input from child processes, it is not possible to get the exact error message. However, the most likely cause of the error, especially when working with Perforce, is that some of the files in the output folder (see the second argument in the error message) are marked as read-only. If that is the cause, you can resolve the error by changing all files to be writable.
+
+Note that you may see similar errors if the same issue applies to the schema generation or the legacy SDK codegen output within the interop code generation step.
+
+------
+
+**Q:** My game uses reliable multicast RPCs - why does the SpatialOS Unreal GDK not support these?
+
+**A:** The underlying implementation of multicast RPCs uses SpatialOS [events](https://docs.improbable.io/reference/13.1/shared/glossary#event) (SpatialOS documentation). SpatialOS events can only be sent unreliably. Additionally, the cost of a multicast RPC scales with the number of clients present in a deployment, which means they can get very expensive. A better approach would be to send RPCs to only the workers that are close to the broadcasting worker.
+
+------
+
+**Q:** I’m getting the error `Unrecognized type 'UCommander' - type must be a UCLASS, USTRUCT or UENUM` when building my project in Visual Studio.
+
+**A:** This is most likely due to not having run the legacy SDK code generation. Try running `Scripts/Codegen.bat` at least once. This should generate the required files for continuing the build process.
+
+------
+
+**Q:** When I try to start SpatialOS from the toolbar plugin, I get the following notification:
+`[improbable.worker.assembly.WorkerAssemblyProvider] The worker assembly does not contain any worker configurations. No workers will be able to connect to this deployment. Unless you specifically intended to start a deployment without any workers, please make sure your assembly was built correctly, and in case of a cloud deployment, also make sure it was uploaded correctly.`
+
+**A:** This is an indication that you haven’t built the worker configurations for the server and the clients. You can fix this by running the `Scripts/BuildWorkerConfig.bat` script. This generates the worker configs which allow your workers to connect to the local instance of SpatialOS.
```

Comment:
Can we change "server" and "clients" to "server worker" and "client workers" here? @danielimprobable 
I would have made the change directly but I wasn't sure if "server worker" needed to be singular, and "client workers" plural, or if they could both be plural.

Reply:
I thought we went away form calling them server worker and client workers as the worker wasn't needed, I think giray corrected that in the original doc. I think we should use plurals for both

Reply:
I think @ElleEss had second thoughts about this and we decided to go for "server-worker" and "client-worker" (yes, including the hyphen, which I'm not convinced by!). Just to be completely explicit. So I'll update these for now, and we can try to push the rest of the company to follow suit.